assets/js/components/user-menu.js

```diff
@@ -160,7 +168,15 @@ class UserMenu extends Component {
 					<Menu
 						ref={ this.menuRef }
 						menuOpen={ menuOpen }
-						menuItems={ [ __( 'Disconnect', 'google-site-kit' ) ] }
+						menuItems={
+							[
+								__( 'Disconnect', 'google-site-kit' ),
+							].concat(
+								proxyPermissionsURL ? [
```

Comment:
I think this also needs to check for `proxyPermissionsURL.length`, because it can be an empty string.

Reply:
Empty strings are considered falsy
![image](https://user-images.githubusercontent.com/1621608/67011105-9fcb4d00-f0f7-11e9-9581-62ab4983143b.png)

Reply:
OMG 🤯 I had no idea :)